### ISSUE TITLE
Talkback should announce roles for CCB buttons

### ISSUE THREAD

**Issue by jasonLiu-jiacli** [microsoft/fluentui-android#369]
Talkback is not announcing roles for CCB buttons.

Repro steps:
1. Open the demo app and turn on Talkback.
2. Go to "ContextualCommandBar"
3. Swipe the focus to any CCB buttons. Talkback is not announcing the role of the button. It should announce "XXX, button".
4. Swipe the focus to other buttons, e.g, "Insert Item". Note that it announces "Insert item, button".

**Comment by Anubhvv** [comment]
> Talkback is not announcing roles for CCB buttons.
> 
> Repro steps:
> 
> 1. Open the demo app and turn on Talkback.
> 2. Go to "ContextualCommandBar"
> 3. Swipe the focus to any CCB buttons. Talkback is not announcing the role of the button. It should announce "XXX, button".
> 4. Swipe the focus to other buttons, e.g, "Insert Item". Note that it announces "Insert item, button".

Hey , I think this Issue doesn't exist anymore, check the screen recording.


https://github.com/microsoft/fluentui-android/assets/68989156/e38bd2f0-09fc-45d1-a72e-9f4595009b63
